config/casestudies/adeo.yml

```diff
@@ -0,0 +1,272 @@
+id: adeogroup
+company:
+  name: Adeo Group
+  description: Adeo owns different brands in retail industry focused on home improvement and DIY markets, like Leroy Merlin.
+  customers: 500M
+  industry: Retail
+  revenue: 25.6B EURO turnover, including 768M EURO online.
+  website: https://www.adeo.com/
+  logo: /img/casestudies/adeo/logo.svg
+  contact:
+    - name: Ludovic Dussart
+      link: https://twitter.com/ldussart
+challenges: |
+    Cost Component Repository product, part of the ADEO tech products, is used to calculate and publish transfer prices between different internal locations on the globe. Different business units use different information systems. It is hard to learn how each business unit share information about their systems, their API and accuracy.
+
+    Initial solution was a developer portal with list of all applications and reference to dedicated documentation. Some legacy systems had docs written in MS Excel.
+
+    There was a need for a standart way of describing event-driven architecture.
+solution: |
+    The API is now described with AsyncAPI. AsyncAPI file, stored together with the source code, is used to generate HTML documentation in the same release pipeline for the product. Documentation is exposed as part of the product internally for other company units depending on the API.
+
+    Payloads are described with Avro schema. These schemas are used to generate models and they are also referenced directly in AsyncAPI files thanks to `schemaFormat` feature and `$ref`. This way they make sure code is aligned with docs.
+
+    Shift to using AsyncAPI also enables the team to implement more use cases using AsyncAPI files.
+technical:
+  languages:
+    - Java
+  frameworks:
+    - Spring
+  protocols:
+    - Kafka
+  brokers: |
+    Kafka with Kafka Connect component. There are 15 production brokers with 47 topics. 
+  testing: |
+    For Kafka e2e tests are done with [Zerocode](https://github.com/authorjapps/zerocode). Load tests are handled with [JMeter](https://jmeter.apache.org/) with [kloadgen](https://github.com/corunet/kloadgen) plugin that supports Kafka and Avro.
+  architecture: |
+    The following [enterprise integration patterns](https://www.enterpriseintegrationpatterns.com/patterns/messaging) are applied:
+    - [Request/Reply](https://www.enterpriseintegrationpatterns.com/patterns/messaging/RequestReply.html)
+      Described with `description` field in AsyncAPI. Reply goes to dedicated reply channel.
+      
+      Example description of response channel:
+      ```
+      description: >
+      This topic is used to REPLY Costing Requests and is targeted by the
+      `REPLY_TOPIC` header.
+      ```
```

Comment:
it seems that carriage return is not properly handled, no ? 
![image](https://user-images.githubusercontent.com/5501911/223370246-70c70348-08e9-478e-a30e-b0b3ae81c2e6.png)

Reply:
good eye, will fix in next commit